### PR TITLE
source tree example.

### PR DESCRIPTION
**Pull request by clemens-tolboom**

### CODE REVIEW

**Comment by Numerlor** [comment]
The packages themselves can provide the entry points through `__main__.py` instead of files in the root, for other files it should be left up to when we get actual code in.
so
```py
root_folder/
client/
├─ __main__.py
├─ __init__.py
server/
├─ __main__.py
├─ __init__.py
```
where client and server are runnable with `python -m server` and `python -m client`

**Comment by clemens-tolboom** [comment]
Thanks ... I did know __init__ but not __main__ and those effects

**Comment by Numerlor** [comment]
the run files won't be necessary with main in place as that serves their purpose. Also are we going to be hosting any html through fastapi? Or is that just for the sake of example?

**Comment by Numerlor** [comment]
I think client is am ore fitting name than gui, as it'll be handling more things around the gui. When we get to it, it will probably be a sub-package of the client

File: server/__main__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI, WebSocket
+from starlette.staticfiles import StaticFiles
+
+"""
+uvicorn server.__main__:app --reload
+
+Run FastAPI server from both web and websocket"""
+app = FastAPI()
```
Comment (Numerlor) [inline server/__main__.py]:
```suggestion
"""
uvicorn server.__main__:app --reload

Run FastAPI server from both web and websocket
"""
app = FastAPI()
```
I think this looks a bit better

Comment (clemens-tolboom) [reply server/__main__.py]: Done and removed 'web'.

**Review by Numerlor — APPROVED** [review]

**Pull request merged**